### PR TITLE
[doc-cws] Removes mention of large tile

### PR DESCRIPTION
**Pull request by AmySteam**
Updating docs to match [CWS announcement](https://developer.chrome.com/docs/extensions/whatsnew/#cws-large-promo-tile) regarding the large tile.

### CODE REVIEW

File: site/en/docs/webstore/best_listing/index.md [hunk 1]
```diff
@@ -140,13 +140,12 @@ Promotional images are your chance to capture users' attention and entice them t
...
+Small promo tile: 440x280 pixels
+: Appears on the homepage, category pages, and in search
 results.
 
-**Large promo tile: 920x680 pixels** &emsp; Not currently used. You do not need to provide a large
-promo tile at this time.
-
-**Marquee image: 1400x560 pixels** &emsp; This image is used if your item is chosen for the marquee
+Marquee image: 1400x560 pixels
+: This image is used if your item is chosen for the marquee
```
Comment (jpmedley) [inline site/en/docs/webstore/best_listing/index.md:148]:
```suggestion
: Used if your item is chosen for the marquee
```

Comment (jpmedley) [reply site/en/docs/webstore/best_listing/index.md:148]: Definitions are one of the few places where sentence fragments are permitted in English. What would be the subject of the sentence is already present in the term being defined. That makes this an application of rule II-17: omit needless words.

File: site/en/docs/webstore/images/index.md [hunk 2]
```diff
@@ -239,31 +238,20 @@ thumb for designing your images:
...
-      </td>
-    </tr>
-  </tbody>
-</table>
+<figure>
+{% Img src="image/nPi5hOhvGtatFrSN6ugwlPVHcs43/9KruJ8pLUNTrzXIXP6JN.png", alt="Small promo image", height="140", width="220" %}
+  <figcaption>
+Small promo image (440x280)
+  </figcaption>
+</figure>
```
Comment (jpmedley) [inline site/en/docs/webstore/images/index.md:246]: I've been meaning to ask you an Ian to start making sure you're using `<figure>` and `<figcaption>`.

File: site/en/docs/webstore/best_listing/index.md (hunk 1 above)
Comment (jpmedley) [reply site/en/docs/webstore/best_listing/index.md:148]: When you run into this, you should also make sure the sentence fragments have similar grammatical structure. That's an application of rule II-19: Express coordinate ideas in similar form.

**Review by jpmedley — CHANGES_REQUESTED** [review]
This looks good. I have one nit and some informational comments.

File: site/en/docs/webstore/best_listing/index.md (hunk 1 above)
Comment (AmySteam) [reply site/en/docs/webstore/best_listing/index.md:148]: Thanks! 👍  Makes sense :)

File: site/en/docs/webstore/images/index.md (hunk 2 above)
Comment (AmySteam) [reply site/en/docs/webstore/images/index.md:246]: Good idea 💯 .

**Review by jpmedley — APPROVED** [review]

**Pull request merged**